exs/mundo_1/javascript/021.html

```diff
@@ -0,0 +1,22 @@
+<html>
+    <title>
+        Audio
+    </title>
+    <body>
+        <h1>Audio</h1>
+        <audio src="radioactive.mp3"></audio>
```

Comment:
Na propriedade `src`, quando for adicionado o diretório com os arquivos binários necessário (áudio, vídeo etc), bastará alterar o path. Mantenha assim até que o diretório seja criado